magento/magento2
# magento/magento2-page-builder#543: Content Product Attributes Default…

### Description (*)
Reordered the `description` and `short_description` product attributes to better accommodate planned Page Builder content staging changes.

### Fixed Issues
1. Fixes magento/magento2-page-builder#543

### Builds
_N/A_

### Related Pull Requests
https://github.com/magento/magento2-page-builder/pull/585


### Questions or Comments
_N/A_

### Checklist

---

### Test Plan
#### Manual Testing Scenarios
- [ ] **New Magento Installation**
1. Create a new Product (**Catalog** » **Products** » **Add Product**)
2. Expand the **Content** accordion
3. Verify that **Short Description** appears _before_ **Description**
- [ ] **Existing Magento Installation**:
1. Create a new Product (**Catalog** » **Products** » **Add Product**)
2. Expand the **Content** accordion
3. Verify that **Description** appears _before_ **Short Description**
4. Update to include this changeset
5. Run `php bin/magento setup:upgrade`
6. Create a new Product (**Catalog** » **Products** » **Add Product**)
7. Expand the **Content** accordion
8. Verify that **Short Description** appears _before_ **Description**
- [ ] **Attribute Sorting in Admin**:
1. Modify Attribute Sets (**Stores** » **Attributes** » **Attribute Set**)
2. Click on the "Default" Attribute Set
3. Switch the order of `description` and `short_description` respectively
4. Click **Save**
5. Create a new Product (**Catalog** » **Products** » **Add Product**)
6. Expand the **Content** accordion
7. Verify that **Description** appears _before_ **Short Description**

#### Automated MFTF Tests
- No new tests needed
-- Scenarios are not high enough priority 
- Update existing tests to pass

#### Manual Hip Tests
- No new tests needed. No updated tests needed
-- Scenarios are not high enough priority
